application/ui/components/params.jsx

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+var _               = require('underscore');
+var React           = require('react');
+var ParamRow        = require('./param-row');
+
+module.exports = React.createClass({
+    displayName : 'Params',
+
+    propTypes : {
+        params   : React.PropTypes.array,
+        onChange : React.PropTypes.func
+    },
+
+    getInitialState : function()
+    {
+        var values   = {};
+        var includes = {};
```

Comment:
Would it make more sense to hold `values` and `includes` as arrays instead of objects? As far as I can see, they are just objects where the `param.name` key is `true`, which essentially makes them just a list.